### ISSUE TITLE
Can't install without peer dependency errors

### ISSUE THREAD

**Issue by finteractive** [dbtedman/postcss-prefixwrap#116]
Hello and thanks for this great plugin

```
yarn add postcss@^8
yarn add postcss-prefixwrap --dev --exact
```
Results in an error 
```
postcss-prefixwrap@1.19.0" has incorrect peer dependency "postcss@7.*.*|^8.1.0".
```

I'm not experienced with postCSS at all but though trial and error installed postcss-prefixwrap@1.17.0which has the  
```
"peerDependencies": {
    "postcss": "*"
  },
```
This works for my use-case but I suspect the "or" in the dependency may be off. 

I'm using yarn 1.22.10

**Comment by dbtedman** [comment]
Hi, thanks for the bug report, and the kind feedback.

Work has been rather busy this week so apologies for the slow response. 

I should be able to get to applying a fix soon and when a new release is available I will let you know.

**Comment by dbtedman** [comment]
@finteractive this issue should be resolved now, I can confirm that installing the newly published 1.22.1 version of postcss-prefixwrap no longer gives a peer dependency warning.
